File: genealogy-tree-API/GenealogyTree.Domain/DTO/Person/PersonDetailsModel.cs
using GenealogyTree.Domain.DTO.Marriage;
using GenealogyTree.Domain.DTO.ParentChild;
using System;
using System.Collections.Generic;

namespace GenealogyTree.Domain.DTO.Person
{
    public class PersonDetailsModel : PersonCreateUpdateModel
    {
        public ICollection<ParentChildModel> Parents { get; set; }
        public ICollection<ParentChildModel> Children { get; set; }
        public ICollection<MarriedPersonModel> Marriages { get; set; }
    }
}

File: gene-tree/src/app/modules/person/models/person/person-details.model.ts
import { RelativeModel } from './../relative/relative.model';
import { MarriedPersonModel } from "../marriage/married-person.model";
import { PersonEditModel } from "./person-edit.model";


export class PersonDetailsModel extends PersonEditModel{
  parents: RelativeModel[];
  children: RelativeModel[];
  marriages: MarriedPersonModel[];
}
